### DISCUSSION TITLE
A large number of Vnodes in skywalking during high concurrency

### DISCUSSION THREAD

**Discussion by wangmengjian** [apache/skywalking#11723] [General]
I used skywalking to track every request of the microservice project, and everything worked fine during low concurrency, but skywalking would show a large number of Vnodes during high concurrency.
![微信图片_20240104112501](https://github.com/apache/skywalking/assets/43922432/f57bccf1-7422-4cae-820f-0060d19cd2d6)

**Comment by wu-sheng** [comment]
This can't be answered easily. We have explained the VNode, and how is determined by your deployment and use case.
